org_fedora_oscap/content_handling.py

```diff
@@ -308,6 +350,7 @@ def get_profiles(self, data_stream_id, checklist_id):
             id_ = OSCAP.xccdf_profile_get_id(profile)
             title = oscap_text_itr_get_text(OSCAP.xccdf_profile_get_title(profile))
             desc = oscap_text_itr_get_text(OSCAP.xccdf_profile_get_description(profile))
+            desc = parse_HTML_from_content(desc)
```

Comment:
minor nitpick: just directly call it instead of saving it in desc and then overwriting desc